index.html

```diff
@@ -4975,6 +4975,621 @@ <h2 id="AudioWorkerParamDescriptor">
+          custom node.
+        </p>
+        <p>
+          An AudioWorklet object provides the capability to import module
+          scripts into its associated <a>AudioWorkletGlobalScope</a>. The user
+          agent can then create <a>AudioWorkletProcessor</a> classes registered
+          on the <a>AudioWorkletGlobalScopes</a> and construct their instances.
+          A processor object represents an audio processing module that runs on
+          the audio <a>rendering thread</a>. After the processor definition is
+          successfully stored, an instance of <a>AudioWorkletNode</a> can be
+          created in the main global scope. This is the counterpart of the
+          processor but lives on the main <a>control thread</a>. These two
+          components can communicate via message passing.
+        </p>
+        <dl title="interface AudioWorklet : Worklet" class="idl">
+          <dt>
+            Promise&lt;void&gt; import(DOMString moduleUrl)
+          </dt>
+          <dd>
+            <p>
+              Imports a module script returns a promise once it is fully loaded.
+            </p>
+          </dd>
+        </dl>
+
+        <h3>
+          Construction Procedure of AudioWorkletNode and AudioWorkletProcessor
+        </h3>
+        <p>
+          The construction procedure of <a>AudioWokletNode</a> and the
+          corresponding <a>AudioWorkletProcessor</a> object consists of several
+          coordinated interactions and the following algorithm specifies such
+          procedure. Note that these steps must be performed in an atomic
+          fashion.
+        </p>
+        <ol>
+          <li>
+            <em>In the window scope</em>: Start AudioWorkletNode constructor.
+          </li>
+          <li>
+            <em>In the AudioWorkletGlobalScope</em>: Locate a registered
+            AudioWorkletProcessor subclass based on name argument. If none is
+            found, or if subclass does not extend AudioWorkletProcessor, throw
+            an exception from the node constructor.
+            <ol>
+              <li>
+                Start registered constructor for custom processor subclass,
+                passing structured clone of the options object that was provided
+                to AudioWorkletNode.
+              </li>
+              <li>
+                Start super constructor for AudioWorkletProcessor.
+              </li>
+              <li>
+                Initialize the processor’s states.
+              </li>
+              <li>
+                End the construction of an AudioWorkletProcessor and return to
+                AudioWorkletNode constructor.
+              </li>
+            </ol>
+          </li>
+          <li>
+            <em>In the window scope</em>: Construct AudioParam objects in the node
+            based on the information from the processor’s
+            parameterDescriptors getter and initialize parameter values from
```

Comment:
parameterDescriptors -> `<code>parameterDescriptors</code>`.  Maybe add a tag if there's a dfn for it.